internal/verification/phaverify_test.go

```diff
@@ -14,7 +14,7 @@
 
 // Package verification provides the ability to verify the diagnosis certificates
 // (JWTs) coming from public health authorities that are responsible for verifying
-// diagnosis pin codes and ceritfying the TEKs.
+// diagnosis pin codes and verifying the TEKs.
```

Comment:
This is not a typo

Reply:
It is a typo `ceritfying -> certifying`. But `verifying` is not correct.